nextflow.config

```diff
@@ -85,6 +84,9 @@ params {
   config_profile_description = false
   config_profile_contact = false
   config_profile_url = false
+  validate_params = true
+  show_hidden_params = false
+  schema_ignore_params = 'genomes,input_paths'
```

Comment:
Probably don't need this?

Also, `input` should technically be `null` if not set by default instead of `false` (though not sure that it makes any difference at this point).

Reply:
As with mhcquant, I think that you can strip all mention of `genomes`, right?